packages/lib/WebDavApi.js

```diff
@@ -441,7 +442,16 @@ class WebDavApi {
 				throw newError(`${message} (Exception ${code})`, response.status);
 			}
 
-			throw newError('Unknown error 2', response.status);
+			let message = 'Unknown error 2';
+			if (response.status === 401 || response.status === 403) {
+				if (!authToken) {
+					message = _('Access denied: Empty password. You may have to re-enter your WebDAV credentials.');
```

Comment:
When we get here does it always mean that the password is empty? In which case, we could simply write "Access denied: Please re-enter your password"?

Reply:
`authToken` can also be `null` if the username is empty. Thus, `'Access denied: Please re-enter your password and/or username'` may make more sense (or alternatively, an extra check for whether the username is empty or the password is empty).

Reply:
Yes that message "Access denied: Please re-enter your password and/or username" sounds good!